ios/Classes/PluginStore.swift

```diff
@@ -0,0 +1,69 @@
+/* Copyright Airship and Contributors */
+
+import Foundation
+import AirshipKit
+
+class PluginStore {
+    
+    private static let defaults = SwiftAirshipPlugin.defaults
+    
+    public static var config: [String: Any]? {
```

Comment:
Not seeing this being used anywhere, either. Maybe we can get rid of it, too?

Reply:
Yeah, that is a leftover from config file setup